shared/local-debug.d.ts

```diff
@@ -15,8 +15,6 @@ export declare const printRPC: boolean
 export declare const printRPCBytes: boolean
 export declare const printRPCWaitingSession: boolean
 export declare const printRPCStats: boolean
-export declare const reduxSagaLogger: boolean
```

Comment:
removing the redux saga logger as i'm likely the only one who really used it and they split it into a different package and the api it uses changed so its not worth fixing right now. if we need it we can look again

Reply:
Is this for logging the actions (like `FsGen.folderListLoad`)? I find them useful sometimes.

Reply:
no, its for logging the internals of saga itself, likely you've never used it